content/ADR-175-remove-assets-from-build-local-addressables.md

```diff
@@ -0,0 +1,99 @@
+   - We don't plan to migrate "Resources"' API to the corresponding one of "Addressables"
+   - We plan to migrate assets currently added to "Resources" to "Addressables" with a proper distribution over time
+
+![image](https://user-images.githubusercontent.com/118179774/214880545-4ecec0c7-6c50-4a57-97dc-946305719ae7.png)
+
+### Addressables Caching
+"Addressables" uses right the same "Caching" system that "Asset Bundles" does. It works out of the box for all the supported platforms. It is possible to disable caching per group, but in our case we will not be doing it.
+
+### Asset Reference
+We are going to utilize ["AssetReference"](https://docs.unity3d.com/Packages/com.unity.addressables@1.21/manual/AssetReferences.html) and "AssetLabelReference" instead of "SerializeField" to properly reference Assets that should be a part of an Addressable without creating a strong reference to them.
+Proper use cases and distribution of such assets in groups will be defined according to the feature production pipeline with performance in mind.
+
+### Addressables Disposal
+Currently, we don't have a robust mechanism for cleaning up memory from assets. We are not going to introduce it this time either. Thus, we will keep all the Addressables loaded in memory. Considering it's the "Local" mode only, it's not gonna be a problem.
+
+### Simulation
+We will use "Simualte Groups" mode for loading Addressables in the Editor. It will help us detect and investigate problems earlier. We will prefer it over "Asset Database" false mode.
+
+Simulate Groups mode (BuildScriptVirtualMode) analyzes content for layout and dependencies without creating asset bundles. Assets load from the asset database through the ResourceManager, as if they were loaded through bundles. To see when bundles load or unload during game play, view the asset usage in the Addressables Event Viewer window (Window > Asset Management > Addressables > Event Viewer).
+
+Simulate Groups mode will help us simulate load strategies and tweak our content groups to find the right balance for a production release.
+
+### Addressables Validation
+"Addressables" inherits an issue of assets duplication from the Asset Bundles System. Unity [thoroughly described the problem and the way of its resolution](https://docs.unity3d.com/Packages/com.unity.addressables@1.21/manual/ManagingAssets.html#asset-and-assetbundle-dependencies).
+"Addressables" provides the ["Analyze Tool"](https://docs.unity3d.com/Packages/com.unity.addressables@1.21/manual/AnalyzeTool.html) and several built-in presets to help detect and/or mitigate such issues.
+
+We have to launch the analysis and fix the issues before pushing Addressables to prevent duplicates and unnecessary implicit references. 
+
+<img alt="image" src="https://user-images.githubusercontent.com/118179774/215037501-46764cb9-6059-45c9-b37d-e2c4571356e1.png">
+
+
+### CI/CD
+"Streaming Assets" directory should be delivered as an artifact within the same package. 
```

Comment:
Do you think its useful to reference that this is done here?

https://github.com/decentraland/unity-renderer/blob/4b383bd926143197baccf0a94dd5a501b61b5b74/browser-interface/scripts/build.ts#L50

Reply:
Can we explain it in words so it's understandable for anybody who takes a look on it for the first time?

Reply:
Done

Reply:
I like the way you put it, thank you @dalkia